war/pom.xml

```diff
@@ -138,24 +138,6 @@ THE SOFTWARE.
       <groupId>org.jenkins-ci.modules</groupId>
       <artifactId>sshd</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.ui</groupId>
-      <artifactId>jquery-detached</artifactId>
-      <version>1.2.1</version>
-      <classifier>core-assets</classifier>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.ui</groupId>
-      <artifactId>bootstrap</artifactId>
-      <version>1.3.2</version>
-      <classifier>core-assets</classifier>
-      <exclusions>
-        <exclusion>
-          <groupId>org.jenkins-ci.ui</groupId>
-          <artifactId>jquery-detached</artifactId>
-        </exclusion>
-      </exclusions>
-    </dependency>
     <dependency>
       <groupId>org.jenkins-ci.ui</groupId>
       <artifactId>handlebars</artifactId>
```

Comment:
It is intentional that this one “core asset” remains?

Reply:
it's used in setup wizard and in plugin manager

Reply:
OK, just checking.